### PR TITLE
fix spk model for resnet, mix update and fix

### PR DESCRIPTION
**Pull request by zh794390558**
**Thank you for submitting a pull request! Please provide the following information for code review:**

# Pull Request Summary
fix speaker_rawmodel for resnet
fix pip install
add stale robot
update hkust egs
formt code

# Test Plan
pushd egs/voxceleb/spk/v1 && bash run_delta.sh

### CODE REVIEW

File: delta/models/speaker_cls_rawmodel.py [hunk 1]
```diff
@@ -169,23 +175,33 @@ def lstm_layer(self, x):
...
         mean, var = tf.nn.moments(x, 1)
         x = tf.concat([mean, tf.sqrt(var + 1e-6)], 1)
     elif pooling_type == 'average':
-      with tf.variable_scope('average_pooling'):
+      with tf.name_scope('average_pooling'):
         mean, _ = tf.nn.moments(x, 1)
         x = mean
     else:
       raise ValueError('Unsupported frame_pooling_type: %s' % (pooling_type))
+    print('pooling layer', x.shape)
```
Comment (hankun11) [inline delta/models/speaker_cls_rawmodel.py]: use `logging`

Comment (zh794390558) [reply delta/models/speaker_cls_rawmodel.py]: will removed.

**Comment by zh794390558** [comment]
PTAL @hankun11

File: delta/utils/loss/loss_utils.py [hunk 2]
```diff
@@ -240,27 +241,28 @@ def arcface_loss(embedding,
         name='arcface_loss_output')
   return output
 
+
 def focal_loss(logits, labels, gamma=2, name='focal_loss'):
-    """
+  """
```
Comment (hankun11) [inline delta/utils/loss/loss_utils.py]: It seems that most part in the repo uses 4 spaces for indentation. Why do you change to use 2 spaces? Either 4 or 2 is fine, as long as we make it consistent ( I personally prefer 4 though :-))
